packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -194,20 +194,27 @@ export default class JSONSchemaBridge extends Bridge {
         .filter(Boolean);
 
       if (combinedPartials.length) {
-        _definition.properties = definition.properties ?? {};
-        _definition.required = definition.required ?? [];
+        const localProperties = definition.properties ?? {};
+        const localRequired = definition.required ?? [];
```

Comment:
With this approach, we'll mutate `definition` and it'd be better not to. How about this:
```suggestion
        const localProperties = definition.properties ? { ...definition.properties } : {};
        const localRequired = definition.required ? definition.required.slice() : [];
```